app/config/Identity.scala

```diff
@@ -13,4 +13,6 @@ class Identity(config: Config) {
   lazy val apiClientToken = config.getString("api.token")
 
   lazy val testUserSecret = config.getString("test.users.secret")
+
+  lazy val useStub = if (config.hasPath("useStub")) config.getBoolean("useStub") else false
```

Comment:
You can add another implicit https://github.com/guardian/support-frontend/blob/103f68ebd78539241340ee36016e106c9edab8df/app/config/ConfigImplicits.scala#L7to create a `getOptionalBoolean`

Reply:
Neat. Will do.